app/views/pages/start.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= content_for :banner do %>
```

Comment:
page title?

Reply:
We discussed this and throught it might be more explicit of what this is being used fall and will rename it to `:start_page_banner`. `:page_title` might be a bit confusing because you might think of the html element `title`